js/languages/en-US.json

```diff
@@ -1363,14 +1362,18 @@
     "provideReview": "Please provide a review.",
     "provideRating": "Please select a rating."
   },
+  "localStorageModelErrors": {
```

Comment:
The model name is localSettings. Can the translation key be consistent with that?